nats-streaming-http-connector/test/producer/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/keda-connectors/nats-streaming-http-connector/test/producer
+
+go 1.13
```

Comment:
Can we have go version and golang image used of same version?

Reply:
changed docker image version